docs/pages/guides/google-authentication.mdx

```diff
@@ -1,329 +1,123 @@
 ---
-title: Authentication with Google and AuthSession API
-sidebar_title: Using Google authentication
-description: A guide on setting up and using Google authentication with AuthSession API in your Expo app.
+title: Using Google authentication
+description: A guide on using @react-native-google-signin/google-signin library to integrate Google authentication in your Expo project.
 ---
 
-import { Terminal } from '~/ui/components/Snippet';
-import { Tabs, Tab } from '~/ui/components/Tabs';
-import { Collapsible } from '~/ui/components/Collapsible';
-import { CODE } from '~/ui/components/Text';
-import { Step } from '~/ui/components/Step';
+import { BoxLink } from '~/ui/components/BoxLink';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 
-AuthSession API allows [browser-based authentication](/versions/latest/sdk/auth-session/#how-web-browser-based-authentication-flows-work) (using OAuth or OpenID Connect) to your project for Android, iOS, and the web. This guide provides steps on configuring and using the API with Google and a [development build](/develop/development-builds/create-a-build/).
+The [`@react-native-google-signin/google-signin`](https://github.com/react-native-google-signin/google-signin) library provides different ways, such as Firebase and Google API, to integrate Google authentication in your Expo app. It also provides native sign-in buttons and supports various authentication methods (standard, server-side validation, and offline access). You can use the library in your project by adding the [config plugin](/config-plugins/introduction/) in the [app config](/versions/latest/config/app/).
 
-## Install libraries
+This guide provides additional information on how to configure the library for your project, and different steps might be required depending on the authentication method you choose.
 
-To use AuthSession API, you'll need to install the following packages in your project:
+## Prerequisites
 
-<Terminal cmd={['$ npx expo install expo-auth-session expo-crypto expo-web-browser']} />
+`@react-native-google-signin/google-signin` requires [custom native code](/workflow/customizing/) and cannot be used with Expo Go. Before proceeding, you can either use:
```

Comment:
```suggestion
The `@react-native-google-signin/google-signin` library can't be used in the Expo Go app because it requires custom native code. Learn more about [adding custom native code to your app](/workflow/customizing/).
```